src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -647,6 +660,16 @@ For major 1.x versions, minimum minor release including it is *1.4.10*.
 For major 2.x versions, minimum minor release including it is *2.1.5*.
 ====
 
+.Additional info on ignoreTimestamps
+[NOTE]
+====
+"ignoreTimestamps" was only added by
+link:https://issues.apache.org/jira/browse/HBASE-24302[HBASE-24302], so it may not be available on
+all released versions.
+Not available in any 1.x versions.
```

Comment:
could we backport it to branch-1? or is there some implementation reason we can't?

Reply:
Probably yes, cherry-pick didn't work because HashTable/SyncTable is still under hbase-server module, in branch-1.

Reply:
I had submitted [this PR](https://github.com/apache/hbase/pull/1693) for backporting this to branch-1. Once it gets committed, I'll update this note here.

Reply:
HBASE-24351 went into branch-1, so updated the note about 1.x supported versions.